### ISSUE TITLE
Adding a language should be possible with keyboard only

### ISSUE THREAD

**Issue by danielbeeke** [danielbeeke/frm#6]
At the moment you can use one ENTER and then you have to use the mouse to apply the language.
I would like to be able to use two times ENTER.
